test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NetCoreProjectTestCase.cs

```diff
@@ -99,41 +101,40 @@ public async Task WithSourceMappingEnabled_InstallPackageFromPMUIFromExpectedSou
     </packageSourceMapping>
 </configuration>");
 
-                using (var testContext = new ApexTestContext(VisualStudio, projectTemplate, XunitLogger, addNetStandardFeeds: true, simpleTestPathContext: simpleTestPathContext))
-                {
-                    VisualStudio.AssertNoErrors();
-
-                    // Act
-                    CommonUtility.OpenNuGetPackageManagerWithDte(VisualStudio, XunitLogger);
-                    var nugetTestService = GetNuGetTestService();
-                    var uiwindow = nugetTestService.GetUIWindowfromProject(testContext.SolutionService.Projects[0]);
-
-                    // The Install action will automatically create a package source mapping to the selected package source if it's missing,
-                    // so select the source which already has a mapping.
-                    uiwindow.SetPackageSourceOptionToSource("PrivateRepository");
-                    uiwindow.InstallPackageFromUI(packageName, packageVersion);
-
-                    // Assert
-                    VisualStudio.AssertNuGetOutputDoesNotHaveErrors();
-                    CommonUtility.AssertPackageReferenceExists(VisualStudio, testContext.SolutionService.Projects[0], packageName, packageVersion, XunitLogger);
-                    Assert.Contains($"Installed {packageName} {packageVersion} from {privateRepositoryPath}", GetPackageManagerOutputWindowPaneText());
-                }
+
+                VisualStudio.AssertNoErrors();
+
+                // Act
+                CommonUtility.OpenNuGetPackageManagerWithDte(VisualStudio, Logger);
+                var nugetTestService = GetNuGetTestService();
+                var uiwindow = nugetTestService.GetUIWindowfromProject(testContext.SolutionService.Projects[0]);
+
+                // The Install action will automatically create a package source mapping to the selected package source if it's missing,
+                // so select the source which already has a mapping.
+                uiwindow.SetPackageSourceOptionToSource("PrivateRepository");
+                uiwindow.InstallPackageFromUI(packageName, packageVersion);
+
+                // Assert
+                VisualStudio.AssertNuGetOutputDoesNotHaveErrors();
+                CommonUtility.AssertPackageReferenceExists(VisualStudio, testContext.SolutionService.Projects[0], packageName, packageVersion, Logger);
+                StringAssert.Contains(GetPackageManagerOutputWindowPaneText(), $"Installed {packageName} {packageVersion} from {privateRepositoryPath}");
             }
         }
 
-        [NuGetWpfTheory(Skip = "https://github.com/NuGet/Home/issues/12898")]
-        [MemberData(nameof(GetNetCoreTemplates))]
+        [Ignore] //https://github.com/NuGet/Home/issues/12898
```

Comment:
Here and everywhere else we use `[Ignore]`, why not this?

```suggestion
        [Ignore("https://github.com/NuGet/Home/issues/12898")]
```

Reply:
Thanks! Fixed.